PubNubUnity/Assets/PubNub/Runtime/Util/UnityWebGLHttpClientService.cs

```diff
@@ -11,6 +12,13 @@ namespace PubnubApi.Unity {
 	/// This is an implementation of the PubNub Transport Layer created for Web GL builds compatibility
 	/// </summary>
 	public class UnityWebGLHttpClientService : IHttpClientService {
+
+		private PubnubLogModule logger;
+
+		public void SetLogger(PubnubLogModule logger) {
```

Comment:
Similar as mentioned [here](https://github.com/pubnub/c-sharp/pull/249/files#diff-e484a935ee9bfbff481bcd6e803cb843421d321aafc299ef26667ff3e7f59d98R12)  we can update this to `IPubnubLogger` by just having replaceAll.